src/Credentials.php

```diff
@@ -1,63 +1,51 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Keboola\OAuthV2Api;
 
+use InvalidArgumentException;
+
 class Credentials extends Common
 {
-    public function __construct($sapiToken, $config = [])
```

Comment:
config je povinnej, protože se musí vždy nastavit URL (stejně se to v reálu dělá)